lib/lightning_web/live/run_live/workorder_component.ex

```diff
@@ -273,28 +277,48 @@ defmodule LightningWeb.RunLive.WorkOrderComponent do
                       </span>
                     <% end %>
                   <% end %>
+                </div>
+                <div role="columnheader" class="col-span-2 px-4">
                   <%= case run.state do %>
                     <% :available -> %>
-                      enqueued @ <.timestamp timestamp={run.inserted_at} />
+                      enqueued @
+                      <.timestamp
+                        tooltip_prefix="Run created at"
+                        timestamp={run.inserted_at}
+                      />
                     <% :claimed -> %>
-                      claimed @ <.timestamp timestamp={run.claimed_at} />
+                      claimed @
+                      <.timestamp
+                        tooltip_prefix="Run claimed by worker at"
+                        timestamp={run.claimed_at}
+                      />
                     <% :started -> %>
-                      started @ <.timestamp timestamp={run.started_at} />
+                      started @
+                      <.timestamp
+                        tooltip_prefix="Run started at"
+                        timestamp={run.started_at}
+                      />
                     <% _state -> %>
-                      <%= run.state %> @ <.timestamp timestamp={run.finished_at} />
+                      finished @
```

Comment:
@stuartc , this isn't on `main`, but only appears to be an issue when the run crashes. i can replicate it on `v2.0.9-pre5` if i set a job to run with only `fn(state => this is a bbbad execption)`. I'm looking at this code here and can't quite figure out why it would behave any different from what's on `main`.

Reply:
Thanks!

Reply:
Ok I've pushed some fixes, looks like all but ☝️ this one is fixed.